modules/custom/az_migration/migrations/migrate_plus.migration.az_paragraph_card_deck.yml

```
@@ -83,14 +86,49 @@ process:
           migration:
             - az_media
 
+  card_width_processed:
+    plugin: default_value
+    default_value: 'col-md-4 col-lg-4'
```

I also believe this is the new default (Cards per row on tablet is set to 1)
```suggestion
    default_value: 'col-lg-4'
```